backend/holon/models/update_register.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+class UpdateRegister(models.Model):
+    """
+    Model describing an User Registration
+    """
+
+    RATING_CHOICES = (
```

Comment:
We hadden tijdens de refinement besproken dat het ook mogelijk moet zijn om alleen een emoticon als reactie te geven zonder details. Dit is in bovenstaande api niet mogelijk, je kan het opsplitsen in 2 endpoints of in 1 api afvangen. Je verliest dan wel de link tussen de rating en de contactgegevens @thesethtruth is dit een probleem?

Reply:
No Link between emoticons and registration